bundles/rust/README.md

```diff
@@ -0,0 +1,33 @@
+# howl-rust
```

Comment:
This README has not been updated to account for inclusion in core - the background history (while interesting), is not relevant for a core bundle IMO, and the `usage` section is obviously not applicable once it's merged.